examples/storm-kafka-examples/pom.xml

```diff
@@ -43,19 +48,19 @@
             <groupId>org.apache.storm</groupId>
             <artifactId>storm-kafka</artifactId>
             <version>${project.version}</version>
-            <scope>${provided.scope}</scope>
+            <scope>${kafka.dependency.scope}</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>${storm.kafka.artifact.id}</artifactId>
             <version>${storm.kafka.version}</version>
-            <scope>${provided.scope}</scope>
+            <scope>${kafka.dependency.scope}</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
             <version>${storm.kafka.version}</version>
-            <scope>${provided.scope}</scope>
+            <scope>${kafka.dependency.scope}</scope>
```

Comment:
Sure, since storm-kafka-client-examples doesn't depend on storm-kafka-examples anymore this isn't important. I'll change it to compile in storm-kafka-client-examples too.